documentation/src/main/asciidoc/topics/con_clients_marshalled_objects.adoc

```diff
@@ -0,0 +1,24 @@
+[id='clients-marshalled-objects_{context}']
+= Clients and marshalled objects
+If you configure cache encoding to store marshalled objects, then you also need to configure clients to operate on those caches.
+
+[discrete]
+== Hot Rod clients
+
+The media type you use to encode caches should match the client marshaller.
+For example, if you configure cache encoding with the `application/x-protostream` media type, you should use the ProtoStream marshaller.
+
+Likewise, if you use `JavaSerializationMarshaller` or `GenericJBossMarshaller` you should encode caches with the corresponding media type.
+
+Additionally, for ProtoStream and Java Serialization marshallers are available by default on {brandname} Server classpath.
+If you use another marshaller, such as JBoss marshalling or a custom implementation, you must add the marshaller JAR to the {brandname} Server classpath.
```

Comment:
jboss marshalling is included by default on the server. Do you mean the client classpath here?

Reply:
hmm. yeah, thanks for catching this. let's just remove as it's duplicated and we explain how to configure JBoss marshalling elsewhere.